core/files/css/base.php

```diff
@@ -327,7 +327,9 @@ public function add_control_rules( array $control, array $controls_stack, callab
 
 		$stylesheet = $this->get_stylesheet();
 
-		$control = apply_filters( 'elementor/files/css/selectors', $control, $value ?? [] );
+		$post_id = $this->get_post_id();
```

Comment:
Make it generic, for example: `$context = $this->get_context()`

```
public function get_context() {
	return null;
}
`

Reply:
I need only the `post_id`, do you think that is a good idea to pass the entire context just to get a specific integer property?

Reply:
I used `$this` and moved the logic to the lazyload module